bsconfig.json

```diff
@@ -1,18 +1,23 @@
 {
-  "name": "reason-cognito",
-  "version": "0.1.0",
-  "sources": {
-    "dir": "src",
-    "subdirs": true
-  },
-  "package-specs": {
-    "module": "commonjs",
-    "in-source": true
-  },
-  "suffix": ".bs.js",
-  "bs-dependencies": ["bs-fetch", "reason-future"],
-  "warnings": {
-    "error": "+101"
-  },
-  "refmt": 3
+	"name": "reason-cognito",
```

Comment:
Can you revert all the changes to this file (`git checkout origin/master bsconfig.json` should do it) and push back up?  It looks like the only changes to this file are formatting and I will eventually enforce that in some way for readability/reviewability.

Reply:
Actually I have no clue how to do this given somehow github is cool with us having the same branch name.... copy + paste probably easier lol.

Reply:
You know what happened here? I was working in a folder off git. Then when i got everything going, I just cp'ed into an older version of the fork, which I see now was stupid. Never even thought about it.